### PR TITLE
fixing delete and some other unit-tests wrt metric store

### PR DESCRIPTION
**Pull request by abaranau**
Follow up on https://github.com/caskdata/cdap/pull/1251, addresses:
* all unit-tests to run

### CODE REVIEW

File: cdap-watchdog/src/test/java/co/cask/cdap/metrics/store/timeseries/FactTableTest.java [hunk 1]
```diff
@@ -123,14 +123,14 @@ public void testBasics() throws Exception {
     expected.clear();
 
     // delete the metrics data at (timestamp + 20) resolution
-    scan = new FactScan(ts + resolution * 2, ts + resolution * 3, null, tagValues);
+    scan = new FactScan(ts + resolution * 2, ts + resolution * 4, null, tagValues);
```
Comment (gsps1) [inline cdap-watchdog/src/test/java/co/cask/cdap/metrics/store/timeseries/FactTableTest.java]: why this change ?

**Comment by gsps1** [comment]
just one question. LGTM. :+1:

File: cdap-watchdog/src/test/java/co/cask/cdap/metrics/store/timeseries/FactTableTest.java (hunk 1 above)
Comment (abaranau) [inline cdap-watchdog/src/test/java/co/cask/cdap/metrics/store/timeseries/FactTableTest.java]: oh

At first I made a change to delete whole rows, so needed to make it *2 since rolltime is 2. Noww I can change it back.

good catch!
